config/locales/pl.yml

```diff
@@ -7,7 +7,17 @@ pl:
     administrator: Administrator
     other: Inni pracownicy edukacyjni
     other_please_specify: Inne (proszę określić)
-    how_using: Jak korzystasz z OpenStax?
+    how_using_book_base: Jak wykorzystujesz
+    how_using_book_options:
+      - Jako główny podręcznik do mojego kursu
+      - Jako opcjonalny/zalecany podręcznik do mojego kursu
+      - Do nauczania, korepetycji lub wsparcia studentów poza ustawieniami kursu
+      - Dla mojej własnej wiedzy lub innej pracy
+    how_using_book_tail: "?"
+    num_students_using_book_base: Ile studentów korzysta z
+    num_students_using_book_time_period: każdego semestru?
+    num_students_using_book_supplemental: Uwzględnij sekcje prowadzone przez asystentów, którymi nadzorujesz
+    num_students_using_book_placeholder: Liczba studentów korzystających z tej książki na semestr
```

Comment:
@ewaforman 👋 If you have some time, would you mind giving these translations a look over? Here are the English versions: https://github.com/openstax/accounts/pull/1193/files#diff-44438ce218f5287c58d0017f965d888715635d94280669896f75841fbd7b4cd7R13 Thank you for your time!

Reply:
@ewaforman Thank you!